# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/moz.build
# Commit: aea7ca0bbd17
# Full Hash: aea7ca0bbd17e31625ac66bbb1815858ff135c35
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-02 21:26:33
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 12: Redirect EarlyHint preload channel to content process r=necko-reviewers,kershaw,valentin
#   
#   Similar to how DocumentLoadListener redirects the document to the content process.
#   
#   This fixes the race conditions in early hint architecture fixing intermittent
# ==============================================================================

diff -r 2fe2afdee09d -r aea7ca0bbd17 netwerk/ipc/moz.build
--- a/netwerk/ipc/moz.build	Fri Dec 02 09:45:28 2022 +0000
+++ b/netwerk/ipc/moz.build	Fri Dec 02 09:45:28 2022 +0000
@@ -41,6 +41,7 @@
     "InputChannelThrottleQueueChild.cpp",
     "InputChannelThrottleQueueParent.cpp",
     "NeckoChild.cpp",
+    "NeckoCommon.cpp",
     "NeckoParent.cpp",
     "NeckoTargetHolder.cpp",
     "ParentChannelWrapper.cpp",